Packs/CyrenInboxSecurity/Integrations/CyrenInboxSecurity/CyrenInboxSecurity.py

```diff
@@ -0,0 +1,494 @@
+    cyren_incidents = client.get_incidents(token, date_from, max_fetch)
+    for ci in cyren_incidents.get('data'):
+        incident_name = 'Cyren Inbox Security - {} ({})'.format(
+            ci.get('threat_type', 'phishing'), ci.get('reported_by', 'System')
+        )
+        incident_timestamp = ci.get('created_at')
+        incident_str_time = timestamp_to_datestring(incident_timestamp * 1000)
+        incident = {
+            'name': incident_name,
+            'occurred': incident_str_time,
+            'rawJSON': json.dumps(ci),
+            'severity': convert_to_demisto_severity(ci.get('confidence', 1)),
+        }
+
+        incidents.append(incident)
+
+        # Update last_fetch if the incident is newer than last_fetch
+        if incident_timestamp > last_fetched_time:
+            last_fetched_time = incident_timestamp + 1
+
+    demisto.setLastRun({
+        'last_fetch': last_fetched_time
+    })
+
+    return incidents
+
+
+def resolve_and_remediate_command(client, args, client_id, client_secret):
+    """
+    Resolve a case and remediate it
+    """
+    case_id = args.get('case_id')
+    resolution = args.get('resolution', 'phishing')
+    resolution_reason = args.get('resolution_reason', '')
+    resolution_reason_text = args.get('resolution_reason_text', '')
+    actions = args.get('actions', [])
+    if actions:
+        actions = actions.split(',')
+
+    if client_id == client_secret == "sample":
+        resp = {"data": {"status": "ok"}}
+    else:
+        token = client.get_token(client_id, client_secret).\
+            get('data').get('access_token')
+        resp = client.resolve_and_remediate(
+            token,
+            case_id,
+            resolution=resolution,
+            resolution_reason=resolution_reason,
+            resolution_reason_text=resolution_reason_text,
+            actions=actions
+        )
+
+    readable_output = (
+        f'{json.dumps(resp)}\n'
+        + '\n*** end of results ***'
+    )
+
+    return CommandResults(
+        outputs=resp,
+        readable_output=readable_output
+    )
+
+
+''' MAIN FUNCTION '''
+
+
+def main() -> None:
+    """
+        PARSE AND VALIDATE INTEGRATION PARAMS
+    """
+
+    instance_name = demisto.integrationInstance()
+    url = demisto.params().get('url')
+    client_id = demisto.params().get('client_id')
+    client_secret = demisto.params().get('client_secret')
+    first_fetch_time = arg_to_datetime(demisto.params().
+                                       get('first_fetch', '3 days').strip())
+    max_results = arg_to_number(arg=demisto.params().get('max_fetch', 50))
+
+    if max_results > MAX_INCIDENTS_TO_FETCH:
+        max_results = MAX_INCIDENTS_TO_FETCH
+
+    try:
+        client = Client(
+            base_url=url,
+            verify=True,
+            proxy=False
+        )
+
+        if demisto.command() == 'test-module':
+            if url == client_id == client_secret == "sample":
+                return_results('ok')
```

Comment:
the "sample" option allows an interested user to run the integration in "sample" mode to see what an incident will look like in XSOAR, without needing to go to Cyren to retrieve an API key or get a CIS license.

Reply:
@Shellyber @ShahafBenYakir - Do we support this use case?

Reply:
After consulting with @ShahafBenYakir - We do can support this use case. 
The only problem is the fetch interval, which is now called like an original fetch every minute.

I think it may be better if you'll do it in a separate command `simulate-fetch-incident`, which means outside of the original fetch, you can run it automatically once the instance is configured, or you can leave it as a normal command to run it manually.

Finally, you need to make sure of the following:

1. In both options (manually and automatically), the command should be run just once, this is so as not to overload too much on XSOAR with dummy data.
2. You need to ensure that if the sample instance is configured, the user will not be able to run the other commands (`fetch-incidents`, `cyren-resolve-and-remediate`).
3. You need to make sure the description is clear for how to use the sample instance.
4. You need to make sure it is clear to the user that the returned data is dummy data and not real (The incident details should contains the word 'dummy' or 'sample').

Reply:
@eran-levy @ditroia - Section 1 and 2 are still not implemented.
Please separate the simulated fetch from the original fetch, so that the simulation will not be repeated.
Also, make sure the user will not be able to run the other commands except the simulate if the "sample" instance is configured.

Reply:
sample fetch is limited to one time. resolve-and-remediate in sample mode can be called and responds with ok. documentation updated to include section on Sample configuration. Sample incident contains the word "sample" in most fields.